src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -0,0 +1,411 @@
+        <div class="flex">
+          <paper-dropdown-menu
+            dynamic-align
+            .disabled=${!item.metadata.writeable}
+          >
+            <paper-listbox
+              slot="dropdown-content"
+              .selected=${item.value}
+              attr-for-selected="value"
+              .key=${id}
+              .property=${item.property}
+              .propertyKey=${item.property_key}
+              @iron-select=${this._dropdownSelected}
+            >
+              ${Object.entries(item.metadata.states).map(
+                ([key, state]) => html`
+                  <paper-item .value=${key}>${state}</paper-item>
+                `
+              )}
+            </paper-listbox>
+          </paper-dropdown-menu>
+        </div>
+      `;
+    }
+
+    return html`${labelAndDescription}
+      <p>${item.value}</p>`;
+  }
+
+  private _isEnumeratedBool(item) {
+    // Some Z-Wave config values use a states list with two options where index 0 = Disabled and 1 = Enabled
+    // We want those to be considered boolean and show a toggle switch
+    const disabledStates = ["disable", "disabled"];
+    const enabledStates = ["enable", "enabled"];
+
+    if (item.configuration_value_type !== "enumerated") {
+      return false;
+    }
+    if (!("states" in item.metadata)) {
+      return false;
+    }
+    if (!(0 in item.metadata.states) || !(1 in item.metadata.states)) {
+      return false;
+    }
+    if (
+      disabledStates.includes(item.metadata.states[0].toLowerCase()) &&
+      enabledStates.includes(item.metadata.states[1].toLowerCase())
+    ) {
+      return true;
+    }
+    return false;
+  }
+
+  private _switchToggled(ev) {
+    this._updateConfigParameter(ev.target, ev.target.checked ? 1 : 0);
+  }
+
+  private _dropdownSelected(ev) {
+    if (ev.target === undefined || this._config![ev.target.key] === undefined) {
+      return;
+    }
+    if (this._config![ev.target.key].value === ev.target.selected) {
+      return;
+    }
+
+    this._updateConfigParameter(ev.target, parseInt(ev.target.selected));
+  }
+
+  private debouncedUpdate = debounce((target) => {
+    const value = parseInt(target.value);
+    this._config![target.key].value = value;
+
+    this._updateConfigParameter(target, value);
+  }, 1000);
+
+  private _numericInputChanged(ev) {
+    if (ev.target === undefined || this._config![ev.target.key] === undefined) {
+      return;
+    }
+    if (this._config![ev.target.key].value === parseInt(ev.target.value)) {
+      return;
+    }
+    this.debouncedUpdate(ev.target);
+  }
+
+  private _updateConfigParameter(target, value) {
+    const data: ZWaveJSSetConfigParamData = {
+      type: "zwave_js/set_config_parameter",
+      entry_id: this.configEntryId!,
+      node_id: this.nodeId!,
+      property: target.property,
+      value: value,
+    };
+
+    if (target.propertyKey !== null) {
+      data.property_key = target.propertyKey;
+    }
+
+    this.hass.callWS(data);
```

Comment:
Don't ever call `callWS` directly. Instead always use functions in `data/zwave_js.ts`

Reply:
This one should still be moved to `data/zwave_js.ts`